client/parties/party-view.tsx

```diff
@@ -129,35 +130,29 @@ function renderPartyMessage(msg: Message) {
   }
 }
 
-interface PartyViewProps {
-  params: { partyId: string }
-}
-
-export function PartyView(props: PartyViewProps) {
+export function PartyView() {
```

Comment:
I don't think removing the params here is a very good idea tbh. If we're putting the party ID in the URL, you should keep that up to date. e.g. if they come to this view and the ID in the URL doesn't match their current party, you should `replace` to the correct URL.

If we're not putting the party ID in the URL, fine, but that doesn't seem to be the case here.